### ISSUE TITLE
Historical NDBC stations

### ISSUE THREAD

**Issue by SorooshMani-NOAA** [oceanmodeling/searvey#161]
In #146 historical stations are not yet implemented. See:

https://github.com/CDJellen/ndbc-api/blob/a4786307869d753e07c3413f1e1c0cf47f418ea2/ndbc_api/ndbc_api.py#L256

**Comment by SorooshMani-NOAA** [comment]
Yes, but the station list that we get right now is only from the active stations, not the historical ones (`stations` vs `historical_stations`).

**Comment by SorooshMani-NOAA** [comment]
Some of these capabilities are very recently added to the upstream NDBC_API code, so it requires the latest version of API to work

**Comment by tomsail** [comment]
ok indeed. 
I think then it'd make sense to pin/limit the version of NDBCApi and merge

**Comment by SorooshMani-NOAA** [comment]
Yes, it's already pinned
